js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -959,6 +989,48 @@ const EditingController = modules.ViewController.inherit((function() {
 
         _beforeUpdateItems: function() { },
 
+        _getVisibleEditColumnIndex: function() {
+            const editColumnName = this.option('editing.editColumnName');
+
+            if(!isDefined(editColumnName)) {
+                return -1;
+            }
+
+            return this._columnsController.getVisibleColumnIndex(editColumnName);
+        },
+
+        _setEditColumnNameByIndex: function(index) {
+            const visibleColumns = this._columnsController.getVisibleColumns();
+            this._setEditColumnName(visibleColumns[index].name, true);
```

Comment:
The null ref exception may be thrown if user pass wrong columnIndex in editCell method

Reply:
Ok